base/cloudbuild.yaml.in

```diff
@@ -0,0 +1,8 @@
+steps:
```

Comment:
Same idea as above, but in this case you're pulling an image from GCR rather than code directly from github, so to keep things consistent it's probably safer to stay on master (since we won't actually be tagging any other images besides `latest`, you'd need to find the specific digest of the image you were using). If we break something then it's easier for us just to rollback rather than have you guys try and work around it, but if you absolutely need to then the option is there.